### PR TITLE
[Merged by Bors] - feat: Limits under division by a positive/negative number

### PR DESCRIPTION
**Pull request by YaelDillies**
and rename a few lemmas to make clear that `neg` refers to `< 0`, not to `-`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/Filter/AtTopBot.lean [hunk 1]
```diff
@@ -1223,6 +1246,12 @@ theorem tendsto_mul_const_atBot_iff [NeBot l] :
   simp only [mul_comm _ r, tendsto_const_mul_atBot_iff]
 #align filter.tendsto_mul_const_at_bot_iff Filter.tendsto_mul_const_atBot_iff
 
+/-- The function `fun x ↦ f x * r` tends to negative infinity along a nontrivial filter
```
Comment (Ruben-VandeVelde) [inline Mathlib/Order/Filter/AtTopBot.lean:1249]:
```suggestion
/-- The function `fun x ↦ f x / r` tends to negative infinity along a nontrivial filter
```

File: Mathlib/Order/Filter/AtTopBot.lean [hunk 2]
```diff
@@ -1265,19 +1306,30 @@ theorem tendsto_mul_const_atBot_iff_neg [NeBot l] (h : Tendsto f l atTop) :
...
 then `f` multiplied by a negative constant (on the right) tends to negative infinity. -/
-theorem Tendsto.atTop_mul_neg_const (hr : r < 0) (hf : Tendsto f l atTop) :
+theorem Tendsto.atTop_mul_const_of_neg (hr : r < 0) (hf : Tendsto f l atTop) :
     Tendsto (fun x => f x * r) l atBot :=
   (tendsto_mul_const_atBot_of_neg hr).2 hf
-#align filter.tendsto.at_top_mul_neg_const Filter.Tendsto.atTop_mul_neg_const
+#align filter.tendsto.at_top_mul_neg_const Filter.Tendsto.atTop_mul_const_of_neg
+
+/-- If a function `f` tends to infinity along a filter,
+then `f` divided by a negative constant (on the right) tends to negative infinity. -/
```
Comment (Ruben-VandeVelde) [inline Mathlib/Order/Filter/AtTopBot.lean:1330]:
```suggestion
then `f` divided by a negative constant tends to negative infinity. -/
```

**Review by Ruben-VandeVelde — APPROVED** [review]
Thanks, please put on the queue after fixing these comments

**Comment by PatrickMassot** [comment]
bors merge

**Pull request closed without merging**